apps/ecs/translations/src/en/fields.json

```diff
@@ -197,7 +197,7 @@
     "label": "Employer telephone number (optional)"
```

Comment:
Change 'Employer telephone number (optional)' to 'Telephone number (optional)' to match figma